src/components/EventCard.tsx

```diff
@@ -39,49 +45,52 @@ const EventCard = ({
-          </div>
-          <img
-            src={MeatballMenuSVG}
-            alt="Menu svg"
-            aria-label="Open Event Card details"
-          />
-        </div>
-        <div className="flex flex-row items-start mb-4">
-          <img src={PinSVG} alt="Pin svg" className="p-1 pt-0" />
-          <span className="text-body-mobile pl-2 pr-12 pt-0.5 font-montserrat break-words w-full">
-            {location}
+    <>
+      <div className="flex my-8 md:my-10">
+        <EventDate startTime={startTime} status={status} />
+        <div className="flex-1 px-6 md:px-10">
+          <span className="font-sans">
+            {startTimeString + (endTime ? " to " + endTimeString : "")}
           </span>
-        </div>
-        <div className="flex flex-row items-start mb-6">
-          <img src={TextIconSVG} alt="TextIcon svg" className="p-1 pt-0" />
-          <p className="text-body-mobile pl-2 pr-12 font-montserrat break-words w-full">
-            {description}
-          </p>
-        </div>
+          <div className="flex flex-row justify-between items-start mb-4">
+            <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
+              {name}
+            </div>
+            <img
+              src={MeatballMenuSVG}
+              alt="Menu svg"
+              aria-label="Open Event Card details"
+            />
+          </div>
+          <div className="flex flex-row items-start mb-4">
+            <img src={PinSVG} alt="Pin svg" className="p-1 pt-0" />
+            <span className="text-body-mobile pl-2 pr-12 pt-0.5 font-montserrat break-words w-full">
+              {location}
+            </span>
+          </div>
+          <div className="flex flex-row items-start mb-6">
+            <img src={TextIconSVG} alt="TextIcon svg" className="p-1 pt-0" />
+            <p className="text-body-mobile pl-2 pr-12 font-montserrat break-words w-full">
+              {description}
+            </p>
+          </div>
 
-        <div className="flex flex-row flex-wrap">
-          {color === "bg-sga-red" ? (
-            <button className="button-base-red px-4 my-2">Vote</button>
-          ) : (
-            <>
-              <button className="button-base-white px-2 my-2 mr-5">
-                Unregister
-              </button>
-              <button className="button-base-red px-4 my-2">See More</button>
-            </>
-          )}
+          <div className="flex flex-row flex-wrap">
+            {status === Status.Live ? (
+              <button className="button-base-red px-4 my-2">Vote</button>
+            ) : (
+              <>
+                <button className="button-base-white px-2 my-2 mr-5">
+                  Unregister
+                </button>
+                <button className="button-base-red px-4 my-2">See More</button>
+              </>
+            )}
+          </div>
         </div>
       </div>
-    </div>
+      <hr className="border-black home-mx" />
```

Comment:
Just something to keep in mind; on the design, there's no line divider between two events if they're on the same day. Not sure what the best way to handle that is; I'm thinking we could perform the logic in the `reduce` on the homepage (add a line divider above the current event if the current event's date is not the same as the previous date's)